client/css/style.css

```diff
@@ -855,13 +854,13 @@ kbd {
 	left: 0;
 	right: 0;
 	top: 50%;
-	border-top: 1px solid #e74c3c;
+	border-top: 1px solid rgba(231, 76, 60, .5);
```

Comment:
Oh, I saw it had changed for the 1 that already had opacity and hadn't noticed it was also added to others. Fine, fine.